preprocessor/src/main/java/com/azure/autorest/preprocessor/tranformer/Transformer.java

```diff
@@ -77,6 +82,40 @@ private void transformSchemas(Schemas schemas) {
     }
   }
 
+  private void transformClients(List<Client> clients, CodeModel codeModel) {
+    for (Client client : clients) {
+      Language language = client.getLanguage().getDefault();
+      Language java = addJavaLanguage(client);
+      java.setName(CodeNamer.toPascalCase(language.getName())); // Name of client should always ends with Client, hence it should not require escaping
+      java.setSerializedName(language.getSerializedName());
+      java.setDescription(language.getDescription());
+      client.getLanguage().setJava(java);
+
+      if (client.getOperationGroups() != null) {
+        for (OperationGroup operationGroup : client.getOperationGroups()) {
+          List<Operation> pagingOperations = new ArrayList<>();
+
+          operationGroup.setCodeModel(client);
+          renameMethodGroup(operationGroup);
+          for (Operation operation : operationGroup.getOperations()) {
+            operation.setOperationGroup(operationGroup);
+
+            if (operation.getExtensions() != null && operation.getExtensions().getXmsPageable() != null) {
+              pagingOperations.add(operation);
+            }
+          }
+
+          // paging
+          for (Operation operation : pagingOperations) {
+            if (nonNullNextLink(operation)) {
+              addPagingNextOperation(client, operation.getOperationGroup(), operation);
+            }
+          }
+        }
```

Comment:
when we have multi-client, we need to update operation group to have correct parent.